package/firmware/layerscape/ls-rcw/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ls-rcw
-PKG_VERSION:=LSDK-20.04
-PKG_RELEASE:=2
+PKG_VERSION:=LSDK-20.04-update-290520
+PKG_RELEASE:=2.1
```

Comment:
PKG_RELEASE is the OpenWrt-internal version.
That means if you update PKG_VERSION, PKG_RELEASE goes back to "1" and is then incremented for "local" changes, until upstream is updated again ...

Reply:
Thanks. Understand now.